types/components/messages.d.ts

```diff
@@ -13,7 +13,12 @@ export interface PayPalMessagesComponentOptions {
             size?: 10 | 11 | 12 | 13 | 14 | 15 | 16;
             align?: "left" | "center" | "right";
         };
+        ratio?: "1x1" | "1x4" | "8x1" | "20x1";
     };
+    placement?: "home" | "category" | "product" | "cart" | "payment";
+    onApply?: Function;
```

Comment:
Let's avoid using `Function` and instead type these out as functions that take in no args and return nothing.
```suggestion
    onApply?: () => void;
```